test/dummy/spec/models/upload_spec.rb

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
+
+require "spec_helper"
+
+describe Goldencobra::Upload do
+  it "can upload a zip file" do
+     zip = Goldencobra::Upload.create(image: File.open(File.join(Rails.root, "spec/fixtures/files/test_archive.zip")))
```

Comment:
`zip = Goldencobra::Upload.create(image: File.open(File.j…` is quite repetitive. Could you find a way to keep the tests working but make it DRYer?